[th2-2064] Fixed the problem in which conn sends a correct user's mes… (#25)
th2-net/th2-conn-generic: conn-fix/README.md

# If this parameter has a negative value (less than ZERO), it won't be used
defaultHeartbeatInterval: 5
```

## Release notes

### 2.4.2

> + Fixed the problem in which conn sends a correct user's message to the remote system but sends an incorrect copy of it to the th2 with a short header instead of its full version

Reviewer: ```suggestion
+ Fixed the problem when the conn sends a correct user's message to the remote system but sends an incorrect copy of it to the th2 with a short header instead of its full version
```

> + Fixed the problem in which conn sends a correct user's message to the remote system but sends an incorrect copy of it to the th2 with short header instead of an enriched version

Reviewer: ```suggestion
+ Fixed the problem in which conn sends a correct user's message to the remote system but sends an incorrect copy of it to the th2 with a short header instead of its full version
```
